lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.ts

```diff
@@ -233,10 +233,32 @@ export class ProcessListCloudComponent extends DataTableSchema implements OnChan
         return !this.rows || this.rows.length === 0;
     }
 
+    /**
+     * Resets the pagination values
+     * @param pagination Pagination values to be set (optional)
+     */
+    resetPaginationValues(pagination?: PaginationModel) {
```

Comment:
Can we use this function just to reset the pagination without taking any parameter? Also can we call it just resetPagination?

Reply:
The optional parameter is necessary since this method is called from _updatePagination_ as well.
I think it's best to handle the reset cases within the same method and the name _resetPaginationValues_ sounds appropriate since we change only the values, unlike _updatePagination_ where we reload the list too.

Reply:
Can we let updatePagination unchanged? I mean, not to call resetPagination inside updatePagination function? 
From the name of the function, resetPagination should only reset the values. Now is also update them if we pass a parameter. It can be confusing

Reply:
Okay. Updated.